hazelcast/src/test/java/com/hazelcast/scheduledexecutor/ScheduledExecutorServiceSlowTest.java

```diff
@@ -153,7 +155,14 @@ public void stats_manyRepetitionsTask()
     public void scheduleRandomPartitions_getAllScheduled_durable()
             throws ExecutionException, InterruptedException {
 
-        HazelcastInstance[] instances = createClusterWithCount(3);
+        ScheduledExecutorConfig scheduledExecutorConfig = new ScheduledExecutorConfig()
+                .setName("s")
+                .setDurability(3);
+
+        Config config = new Config();
+        config.addScheduledExecutorConfig(scheduledExecutorConfig);
+
+        HazelcastInstance[] instances = createClusterWithCount(4, config);
```

Comment:
It's 4 instances now, since `setDurability(3)` requires 1 owner + 3 replicas, correct?